WebInstall/explainer.md

```diff
@@ -112,61 +156,79 @@ Additionally for developers, the feature also:
 
-1. `navigator.install(<url>)`: The method receives a parameter which is a url to a web app to install. This can be a cross-domain app and will prompt for installation of the app if the requesting origin has installation permissions (see security section). This is the most common use of the API.
+2. `navigator.install(<params>)`: The method receives an object with parameters that it can use to customise a same domain installation. These paramters can be used to hint to the UA that the app should be installed in a surface like a side panel.
 
-2. `navigator.install(<url [, <params]>)`: This signature of the method includes the optional parameters. These parameters alter how the app is installed and are defined in an object. More information about the parameters is found in the Params subsection of this specification.
+3. `navigator.install(<url>)`: The method receives a parameter which is a url to a web app to install. This can be a cross-domain app and will prompt for installation of the app if the requesting origin has installation permissions (see security section). This is the most common use of the API.
+
+4. `navigator.install(<url, <params>)`: This signature of the method includes the optional parameters. These parameters alter how the app is installed and are defined in an object. More information about the parameters is found in the Params subsection of this specification.
 
 #### Parameters
 
 The `navigator.install` call can receive an object with a set of parameters that specify different installation behaviours for the app.
 
-* **mode**: Due to the evolving nature of PWAs, there are different surfaces where these can be installed. The optional parameters that can be passed on to the `navigator.install` call can hint to the UA if the app should be installed as a standalone app or as a sidebar bar if supported. `mode` is an array of values that hint to the preferred way of installing the app. Among the values it can have are `default` and `sidebar`. These values will always fallback to a standalone installation.  
+* **mode**: Due to the evolving nature of PWAs, there are different surfaces where these can be installed. The optional parameters that can be passed on to the `navigator.install` call can hint to the UA if the app should be installed as a standalone app or as a sidebar bar if supported. `mode` is an array of values that hint to the preferred way of installing the app. Among the values it can have are `default` and `side_panel`. These values will always fallback to a standalone installation.  
 
 As an example, you could try to install the elk PWA in a browser that supports a sidebar with this code:
 
 ```javascript
-navigator.install('https://elk.zone', {mode: ['sidebar']});
+navigator.install('https://elk.zone', {mode: ['side_panel']});
 ```
+* **referral-info**: this parameter takes the form of an object that can have arbitrary information required by the calling installation domain. 
```

Comment:
Should also consider what the API should do if a caller adds properties to `referral_info` that it's not allowed to modify (like `installSource`, `installTimestamp`, `httpReferrer`, etc...).